Makefile

```diff
@@ -175,7 +175,11 @@ test_doc_test: compile
 test_stdlib: compile
 	@ echo "==> elixir (exunit)"
 	$(Q) exec epmd & exit
-	$(Q) cd lib/elixir && ../../bin/elixir -r "test/elixir/test_helper.exs" -pr "test/elixir/**/*_test.exs";
+	$(Q) if [ $(OS) = Windows_NT ]; then \
+		cd lib/elixir && cmd //C call ../../bin/elixir.bat -r "test/elixir/test_helper.exs" -pr "test/elixir/**/*_test.exs"; \
+	else \
+		cd lib/elixir && ../../bin/elixir -r "test/elixir/test_helper.exs" -pr "test/elixir/**/*_test.exs"; \
+	fi
```

Comment:
@chyndman I don't quiet understand why this is needed in windows, but... wouldn't it be necessary [in the other tests](https://github.com/chyndman/elixir/blob/84b1f6546bc2e51fd1fc53d3d7268d7beb788b54/Makefile#L173) as well, in this doesn't get executed by calling `make test_iex`  for example.